modules/Header/ui/CategoriesNav/CategoriesNavMobile.module.scss

```diff
@@ -0,0 +1,29 @@
+.container {
+    margin: 0 $grid-spacing-mobile;
+    display: none;
+
+    @include mobile-only {
+        display: block;
+    }
+}
+
+.categoryButton {
+    padding: $spacing-2 0 !important;
+}
+
+.divider {
+    border: none;
+    border-top: 1px solid $color-borders;
+    margin: $spacing-2 0;
+}
+
+.featuredContainer {
+    display: flex !important;
```

Comment:
Why using the important here? There's no other rule.

Reply:
70801e12